SDK/Store/StoreViewController.swift

```diff
@@ -18,7 +18,36 @@ class StoreViewController: UIViewController {
     var subscriptionIAP: SubscriptionIAP?
     var product: SKProduct?
     
+    //Properties
+    enum TosLinks {
+        case terms
+        case privacy
+        
+        var text: String {
+            get {
+                switch self {
+                case .terms:
+                    return "Terms of Service"
+                case .privacy:
+                    return "Privacy Policy."
```

Comment:
Why is there a dot in line 32 but not in 30?

Reply:
Fixed, removed the dot, it was there as a left over, because the full text is "Terms of Service and Privacy Policy." The dot is in line 133